src/app/app.component.ts

```diff
@@ -1,4 +1,8 @@
 import { Component } from '@angular/core';
```

Comment:
move logic from app component to question component and answer component

Reply:
done